micrometer-registry-influx/deployment/src/main/java/io/quarkiverse/micrometer/registry/influx/deployment/InfluxRegistryProcessor.java

```diff
@@ -36,9 +38,20 @@ public boolean getAsBoolean() {
     }
 
     @BuildStep(onlyIf = { NativeBuild.class, InfluxRegistryEnabled.class })
-    public MicrometerRegistryProviderBuildItem nativeModeNotSupported() {
-        log.info("The InfluxDB meter registry does not support running in native mode.");
-        return null;
+    public MicrometerRegistryProviderBuildItem nativeModeNotSupported(
+            BuildProducer<AdditionalBeanBuildItem> additionalBeans,
+            BuildProducer<ReflectiveClassBuildItem> reflectionClasses) {
+        log.warn("native support for InfluxDB meter registery is EXPERIMENTAL!!! check reflectionClasses");
+
+        reflectionClasses.produce(new ReflectiveClassBuildItem(true, true, InfluxApiVersion.class));
+
+        // Add the InfluxDB Registry Producer
```

Comment:
You should be able to remove the "onlyIfNot" clause off of the next method.

Constrain this method adding the reflection classes required for native..